doc/content/getting_started/RunningBlackBear.md

```diff
@@ -72,17 +81,17 @@ cd ~/projects/blackbear/
 git pull
 ```
 
-Then update the MOOSE submodule:
+Then update MOOSE:
 
 ```bash
-cd ~/projects/blackbear/
-git submodule update
+cd ~/projects/moose/
+git pull
```

Comment:
Are we recommending git pull or git rebase upstream/{devel|master}? I had thought that the moose guys were recommending git rebase, but I could be out of the loop these days

Reply:
Done